crates/stc_ts_type_checker/tests/tsc.rs

```diff
@@ -335,6 +336,8 @@ fn do_test(file_name: &Path, spec: TestSpec, use_target: bool) -> Result<(), Std
             // Install a logger
             let _guard = testing::init();
 
+            checker.load_typings(&file_name, None, Some(&lib_files));
```

Comment:
But this is problematic I think

Reply:
We have to store those lib files and check for loading error

Reply:
Added lib files.

Reply:
The checker is loading them now, but it still fails at the `import React = require('react');` line even after loading libs.

Can you tell me where I need to look to skip the require if libs are loaded?

I searched for a while but couldn't figure out myself.

Reply:
I thought more about this, and I concluded such checks are not required because if those are not loaded, the test will fail with import errors, sorry for the confusion!

Reply:
Thanks for clarifying, in that case I guess it's ready for review.